Automattic/jetpack
# Launchpad: Update copy for global styles in plan_selected task

Related to https://github.com/Automattic/wp-calypso/pull/82524

## Proposed changes:

* In wpcom, update the copy `custom` to `premium` styles in the context of global styles.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this patch to your sandbox.
* Head to https://developer.wordpress.com/docs/api/console/ and select WP REST API → wpcom/v2.
* Call the endpoint `/sites/${ SITE_SLUG }/launchpad` with a site that has global styles gated (just select any theme with a non-default style variation).
* Ensure that the checklist item `plan_selected` the subtitle has changed to "Your site contains premium styles. Upgrade now to publish them and unlock tons of other features."
